pos_require_product_quantity/models/pos_config.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    require_product_quantity = fields.Boolean(
```

Comment:
If this module is installed, we could imagine that the default behaviour is to forbid empty lines. So i think default= true is more intuitive. Don't you think ?

Reply:
In general my preference is to go with defaults that behave as the standard avoiding possible future weird interactions with other modules tests.

Reply:
Indeed even if i don't see which test could be broken, enabling that feature. Do you have some ecemple ?

But well, to be sure we can :
Set default = true here
Set the value to false in the demo data.

Reply:
It's just a rule of thumb so you don't have to worry thinking about wich cases could break in the future :sweat_smile: (for expample, a Tour changed upstream that suddenly breaks the branch CI)

Reply:
OK. 
but what about my proposal ?

> Set default = true here
> Set the value to false in the demo data.

Reply:
Ok, fine :)

Reply:
I have bad experience with setting default to true (it is not recognisez and stays false) and setting config values in the demo data (unable to update the module) so I'd prefer to keep this like this. I also agree with @chienandalu.